README.md

```diff
@@ -1,6 +1,6 @@
 ### Hi there 👋
 
-My Name is Ajesh R Pai, Working at Thumbworks
+My Name is Ajesh R Pai, Working at Capital Float
```

Comment:
Missing period at the end of line

Reply:
`.` This is not the scope of this PR as it needs investigation from @AjeshRPai.
This PR is responsible for fix which `main` branch had.
Your thoughts @AjeshRPai ?

Reply:
@AjeshRPai I agree with @SachinKasaraddi and I think this needs to be fixed ASAP.